pkg/cmd/cli/install/install.go

```diff
@@ -291,10 +300,18 @@ func (o *InstallOptions) Validate(c *cobra.Command, args []string, f client.Fact
 		return errors.Errorf("Bucket names cannot begin with a dash. Bucket name was: %s", o.BucketName)
 	}
 
-	if o.ProviderName == "" {
+	if o.ProviderName == "" && !o.NoDefaultBackupLocation {
 		return errors.New("--provider is required")
 	}
 
+	if o.NoDefaultBackupLocation && o.ProviderName != "" {
+		return errors.New("Cannot use both --provider and --no-default-backup-location at the same time")
+	}
+
+	if o.NoDefaultBackupLocation && o.BucketName != "" {
+		return errors.New("Cannot use both --bucket and --no-default-backup-location at the same time")
+	}
+
```

Comment:
I think this code would be a little simpler if we structured it like:

```
if o.NoDefaultBackupLocation {
    // ensure ProviderName and BucketName not specified
} else {
    // validate BucketName, ProviderName specified correctly
}
```

Also, we probably also want to validate that if `o.NoDefaultBackupLocation` is true, that `--prefix` and `--backup-location-config` were *not* specified.

Reply:
This applies to L303-L314

Reply:
Thanks. I will take a look at it and try to modify it.

Reply:
Hello, Kriss. I already modified that part of the code you mentioned. Please feel free to take a look at that, thanks.

Reply:
Thanks @Frank51, the code changes look good.